### PR TITLE
test(vitest): use vitest instead of jest

### PR DESCRIPTION
**Pull request by philibea**

### CODE REVIEW

File: .github/workflows/check-pull-request.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4.0.1
         with:
-          node-version: 18
```
Comment (Codelax) [inline .github/workflows/check-pull-request.yml:16]: node 18 is still a major available version. We might want to test it

Comment (philibea) [reply .github/workflows/check-pull-request.yml:16]: I know it's still in the matrix, i changed only the version of node used to fetch node_modules

Comment (Vinzius) [reply .github/workflows/check-pull-request.yml:16]: quickly discussed with @Codelax ; we suggest to change in the matrix 19 to 20, and lgtm :)

File: .eslintignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 dist/
 examples/
+vite.config.ts
```
Comment (Vinzius) [inline .eslintignore:4]:
```suggestion
vite.config.ts

```

**Review by Vinzius — APPROVED** [review]

**Pull request merged**